Sources/Charcoal/CharcoalViewController.swift

```diff
@@ -342,4 +373,9 @@ private extension UserDefaults {
         get { return bool(forKey: #function) }
         set { set(newValue, forKey: #function) }
     }
+
+    var polygonSearchCalloutShown: Bool {
+        get { return bool(forKey: #function) }
```

Comment:
Please prefix this with "Charcoal." like `regionReformCalloutShown`, all userdefaults usage should be prefixed when in a framework. Unfortunately we made a mistake with that for `bottomButtomCalloutShown `.